Fix "Raster layer unique values report" area unit of measure (#9257)
qgis/QGIS-Documentation: docs/user_manual/processing_algs/qgis/rasteranalysis.rst

Raster layer unique values report
---------------------------------

> Returns the count and the area, expressed in the area unit of the layer's CRS, of

Reviewer: I had to read twice to get the meaning. How about, trying to simplify:
```suggestion
Returns the count and the area (in the unit of the layer's CRS) of
```
OK. It is not really the crs unit but i hope people know that `m` is not an area unit.
Or `(based on the unit of the layer's CRS)`?

Author: I'm not a native English speaker... "the area unit of the layer's CRS" is also used in the description of the output table. Why here is not understandable? Maybe we could use the same wording as proposed the description of the output table:
`Returns the count and area of each unique value in a given raster layer. The calculation of the area is done in the area unit of the layer's CRS.`

Reviewer: Good remark re area.
And +1 for splitting the sentence in two.

> Returns the count and the area, expressed in the area unit of the layer's CRS, of
> each unique value in a given raster layer.

Author: ```suggestion
Returns the count and area of each unique value in a given raster layer.
The calculation of the area is done in the area unit of the layer's CRS.
```

       * *value*: pixel value
       * *count*: count of pixels with this value
       * *m2* or *deg2* or *ft2* or ... : total area of pixels with this value.

>          The column name depends on the area unit of the layer's CRS.

Reviewer: And adding some _captain obvious_ comment?
```suggestion
         The column name depends on the area unit of the layer's CRS
         and the calculation is done in that unit.
```

Author: I think the algorithm should be improved adding a string output which will contain the actual name of such column.
